### PR TITLE
chore(deps): upgrade knex and objection

### PR DESCRIPTION
**Pull request by traviscrist**
## Changes proposed in this pull request

- Upgrades knex to 0.95 and objection to 3.0.1
- Follows knex guide at https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: packages/auth/src/access/service.test.ts [hunk 1]
```diff
@@ -121,7 +121,8 @@ describe('Access Service', (): void => {
   test("Doesn't create anything if an empty array is passed", async (): Promise<void> => {
```
[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/auth/src/access/service.test.ts:121]: It looks like `accessService.createAccess` could also enforce a non-empty array by changing the argument type to `[AccessRequest, ...AccessRequest[]]`
https://bobbyhadz.com/blog/typescript-non-empty-array#declare-a-non-empty-array-in-typescript

[the remark this answers is not in the document]
Comment (traviscrist) [reply packages/auth/src/access/service.test.ts:121]: @njlie based on the comment from @wilsonianb should I remove it or not?

[the remark this answers is not in the document]
Comment (njlie) [reply packages/auth/src/access/service.test.ts:121]: Yeah I think you could remove it

[the remark this answers is not in the document]
Comment (traviscrist) [reply packages/auth/src/access/service.test.ts]: @njlie @wilsonianb removed the test, can you please take a look?

**Review by wilsonianb — APPROVED** [review]

File: packages/auth/src/tests/app.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import Knex from 'knex'
+import { Knex } from 'knex'
```
Comment (njlie) [inline packages/auth/src/tests/app.ts:1]: It seems like the import you've been doing in other places is in lowercase, does this import still work?

File: packages/auth/src/index.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { EventEmitter } from 'events'
 import createLogger from 'pino'
-import Knex from 'knex'
+import { knex } from 'knex'
```
Comment (njlie) [inline packages/auth/src/index.ts:3]: Should it be uppercase `Knex` here?

File: packages/auth/src/tests/app.ts (hunk 2 above)
Comment (traviscrist) [reply packages/auth/src/tests/app.ts:1]: yes unfortunately, see: https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950

File: packages/auth/src/index.ts (hunk 3 above)
Comment (traviscrist) [reply packages/auth/src/index.ts:3]: Ditto: https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950

**Review by njlie — APPROVED** [review]
LGTM

**Pull request merged**